perception_pipeline_config:
  scene_bounds:
    min: [-4.0, -2.5, -4.0]
    max: [4.0, 2.5, 4.0]
  voxel_props:
    cubic_size: 1.0 # in metres
    voxel_resolution: 15 # voxels in 'cubic_size'
    primitive_sphere_radius: 0.01 # voxels in 'cubic_size'
